### PR TITLE
add faceted plot

### PR DESCRIPTION
**Pull request by cyouh95**
Test review changes.

### CODE REVIEW

File: pull_request.R [hunk 1]
```diff
@@ -29,4 +29,19 @@ ggplot(data = df_boulder, mapping = aes(x = total_students, y = avgmedian_inc_25
...
-dev.off()
\ No newline at end of file
+dev.off()
+
+#6. Facet the previous subplot by school type
+
+getwd()
+list.files()
+#open png file  
+png("plots/scatterplot_facet.png")
```
Comment (cyouh95) [inline pull_request.R:39]: Change the scatterplot name. (`Add single comment`)

File: pull_request.R [hunk 2]
```diff
@@ -29,4 +29,19 @@ ggplot(data = df_boulder, mapping = aes(x = total_students, y = avgmedian_inc_25
...
+
+#create plot
+ggplot(data = df_boulder, mapping = aes(x = total_students, y = avgmedian_inc_2564, color = as.factor(visited))) + geom_point() +
+  xlab("Total students") + ylab("Avg. median income") + scale_color_discrete(name = "Legend", labels = c("No visit", "Visits")) +
+  facet_wrap(~ school_type)
```
Comment (cyouh95) [inline pull_request.R:44]: Commenting as `Start a review`

File: pull_request.R [hunk 3]
```diff
@@ -29,4 +29,19 @@ ggplot(data = df_boulder, mapping = aes(x = total_students, y = avgmedian_inc_25
...
+
+#close file
+dev.off()
```
Comment (cyouh95) [inline pull_request.R:47]: Another comment (Now the options is only `Add review comment`)

**Review by cyouh95 — COMMENTED** [review]
Here's a general comment (`Review changes` > `Comment`)

File: pull_request.R [hunk 4]
```diff
@@ -29,4 +29,19 @@ ggplot(data = df_boulder, mapping = aes(x = total_students, y = avgmedian_inc_25
...
 
 #close file
-dev.off()
\ No newline at end of file
+dev.off()
+
+#6. Facet the previous subplot by school type
+
+getwd()
+list.files()
```
Comment (cyouh95) [inline pull_request.R:37]: Adding another `Add single comment`

File: pull_request.R [hunk 5]
```diff
@@ -29,4 +29,19 @@ ggplot(data = df_boulder, mapping = aes(x = total_students, y = avgmedian_inc_25
   xlab("Total students") + ylab("Avg. median income") + scale_color_discrete(name = "Legend", labels = c("No visit", "Visits"))
 
 #close file
-dev.off()
\ No newline at end of file
+dev.off()
```
Comment (cyouh95) [inline pull_request.R:32]: Another `Add single comment`

**Review by cyouh95 — COMMENTED** [review]
?

**Review by ozanj — CHANGES_REQUESTED** [review]
change scatterplot color to school_type and change facet_wrap to "visited"

**Pull request merged**